bootstraps/stage2/spack.in.yaml

```diff
@@ -0,0 +1,52 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - glibc %gcc@=13.2.0 arch=linux-spack-x86_64 ^linux-headers%gcc@12
+  - binutils %gcc@=13.2.0 arch=linux-spack-x86_64
+  - gcc@13.2.0 %gcc@=13.2.0 ~bootstrap ~graphite +binutils arch=linux-spack-x86_64
+  - gmp +stage1 arch=linux-spack-x86_64
+  # we have no C++ right now, disable
+  - berkeley-db ~cxx ~stl
+  - ncurses ~cxx
+  config:
+    install_root:
+      padded_length: 256
+  view: true
+  concretizer:
+    unify: true
+  packages:
+    all:
+      target: [x86_64]
```

Comment:
Those `packages:all:target` should probably be changed to `packages:require:one_of:[x86_64,aarch64]`

Reply:
Interesting, I’ll try that thanks.

Reply:
Worked great!